mtgjson5/resources/card_enrichment.json

```diff
@@ -0,0 +1,80 @@
+{
+  "_comment": "This file enriches card data not available from Scryfall. Lookup by UUID (preferred) or by set code with 'number|name' or 'number' as keys. Used to add specific neon ink colors to promo_types.",
```

Comment:
I'd rather we not have comments directly in-line with JSON as it's not well supported at this point and leads to confusion in the parsing code